<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QMainWindow" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск аномалий</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Данные</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Поиск аномалий</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Результаты</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>Открыть...</string>
     </property>
     <addaction name="action_PCAP"/>
     <addaction name="action_Text"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuAnomaly">
    <property name="title">
     <string>Аномалии</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionROC"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnomaly"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFind">
   <property name="text">
    <string>Обнаружить</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_PCAP">
   <property name="text">
    <string>Файл PCAP</string>
   </property>
  </action>
  <action name="action_Text">
   <property name="text">
    <string>Текстовый файл</string>
   </property>
  </action>
  <action name="actionROC">
   <property name="text">
    <string>Вывод результатов</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
